src/algorithms/distancegradient.jl

```diff
@@ -5,15 +5,15 @@ The `sources` field is the *linear* indices of the matrix, from which the
 distance must be calculated.
 """
 struct DistanceGradient <: NeutralLandscapeMaker
-    sources::Vector{Integer}
+    sources::Vector{Int}
```

Comment:
I guess we could have a constructor taking anything iterable and collects it to a `Vector{Int}`?

Reply:
yeah this should accept CartesianIndices, good call